druid/src/widget/label.rs

```diff
@@ -175,6 +180,14 @@ impl<T: TextStorage> RawLabel<T> {
         self
     }
 
+    /// Builder-style method for making this label into a control-label.
+    ///
+    /// If this label is a control-label it will change its color when disabled.
+    pub fn control_text(mut self) -> Self {
```

Comment:
ditto, I'd go with something more explicit maybe? like `control_component` or even `dims_when_disabled`?


Did you consider having labels *always* draw themselves dimmed when they're disabled? I could imagine there maybe being some downsides to this but they aren't totally clear to me..

Reply:
> Did you consider having labels always draw themselves dimmed when they're disabled? I could imagine there maybe being some downsides to this but they aren't totally clear to me..

I don't have a strong opinion how to do this, if you want i change it. If we do this we could at least have the option to disable the color change, but enabling it by default.